crates/rooch/src/commands/object.rs

```diff
@@ -48,3 +75,181 @@ impl CommandAction<Option<StateView>> for ObjectCommand {
         Ok(resp)
     }
 }
+
+#[derive(Parser)]
+pub struct QueryObjectCommand {
+    #[clap(subcommand)]
+    filter: ObjectStateFilterOptions,
+}
+
+#[derive(Parser)]
+pub struct QueryObjectOptions {
+    /// Max number of items returned per page
+    #[clap(long)]
+    limit: Option<usize>,
+
+    /// descending order
+    #[clap(short = 'd', long, default_value = "false")]
+    descending_order: bool,
+
+    /// Render and return display fields.
+    #[clap(long, default_value = "false")]
+    show_display: bool,
+
+    #[clap(flatten)]
+    pub(crate) context_options: WalletContextOptions,
+}
+
+/// Query object by filter
+#[derive(Parser)]
+pub enum ObjectStateFilterOptions {
+    /// Query by object value type and owner.
+    #[clap(name = "type-owner")]
+    ObjectTypeWithOwner {
```

Comment:
it's ok

Reply:
Add the three options to the original command, or add them to the `object query` command?
Is a query command needed?

Reply:
Because the `object query` also supports getting objects by id, it seems like the `object get` command is useless; we can merge the `object get` and `object query` to the `object` command.